engine-standalone-storage/src/sync/mod.rs

```diff
@@ -0,0 +1,233 @@
+use aurora_engine::{connector, engine, parameters};
+use aurora_engine_sdk::{
+    env::{self, Env},
+    promise,
+};
+use aurora_engine_types::TryFrom;
+use borsh::BorshDeserialize;
+
+pub mod types;
+
+use types::{Message, TransactionKind};
+
+pub fn consume_message(storage: &mut crate::Storage, message: Message) -> Result<(), error::Error> {
+    match message {
+        Message::Block(block_message) => {
+            let block_hash = block_message.hash;
+            let block_height = block_message.height;
+            let block_metadata = block_message.metadata;
+            storage
+                .set_block_data(block_hash, block_height, block_metadata)
+                .map_err(crate::Error::Rocksdb)?;
+            Ok(())
+        }
+
+        Message::Transaction(transaction_message) => {
+            // Failed transactions have no impact on the state of our database.
+            if !transaction_message.succeeded {
+                return Ok(());
+            }
+
+            let signer_account_id = transaction_message.signer;
+            let predecessor_account_id = transaction_message.caller;
+            let relayer_address = aurora_engine_sdk::types::near_account_to_evm_address(
+                predecessor_account_id.as_bytes(),
+            );
+            let transaction_position = transaction_message.position;
+            let near_tx_hash = transaction_message.near_tx_hash;
+            let block_hash = transaction_message.block_hash;
+            let block_height = storage.get_block_height_by_hash(block_hash)?;
+            let block_metadata = storage.get_block_metadata(block_hash)?;
+            let current_account_id = "aurora".parse().unwrap();
```

Comment:
This is hard coded, probably should be a constant for now. Would be nice in the future to have configuration settings where this can be changed.

Reply:
Factored out as a constant in https://github.com/aurora-is-near/aurora-engine/pull/382/commits/f8b1611c75eb26dd65de024b1830d906b07f8f49

I agree having a config would be a little nicer. Not necessary for us right now because both testnet and mainnet use the `aurora` account, so I'll leave this for the time being. When we tie everything together in the in the standalone engine binary we will need some configuration for sure (eg storage directory, NATS URL, etc). At that point we can maybe include this kind of "environment" configuration too.